src/Microsoft.ML.PCA/PCACatalog.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms.Projections;
+
+namespace Microsoft.ML
+{
+    public static class PcaCatalog
+    {
+
+        /// <summary>Initializes a new instance of <see cref="PrincipalComponentAnalysisEstimator"/>.</summary>
```

Comment:
></summary> [](start = 101, length = 10)

Not needed.

Reply:
why?  All the examples close the tag:

[https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/xml-documentation-comments](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/xml-documentation-comments)

---
In reply to: [230154186](https://github.com/dotnet/machinelearning/pull/1494#discussion_r230154186) [](ancestors = 230154186)